omero/sysadmins/server-upgrade.txt

```diff
@@ -260,17 +262,34 @@ Your memory settings should be copied along with :file:`etc/grid/config.xml`,
 but you can check the current settings by running :omerocmd:`admin jvmcfg`.
 See :ref:`jvm_memory_settings` for more information.
 
+OMERO.web depedences
+""""""""""""""""""""
+
+The FastCGI support was removed in OMERO 5.2 and OMERO.web can be deployed
+using WSGI :ref:`unix_omero_web_deployment`. If you already deployed
+OMERO.web using WSGI it is highly recommended that you regenerate your
+:ref:`unix_omero_web_deployment`, remembering to merge in any of your own
+modifications if necessary. See :doc:`/sysadmins/whatsnew` for details
+of changes.
+
+If necessary ensure you have set up a regular task to clear out any stale
+OMERO.web session files as described in :ref:`unix_omero_web_maintenance`
+or :ref:`windows_omero_web_maintenance`.
+
+
 OMERO.web server configuration
 """"""""""""""""""""""""""""""
 
-The generated web-server configurations for Nginx and Apache have been
-revised in OMERO 5.1. It is highly recommended that you regenerate your
-:ref:`wsgi_configuration`, remembering to merge in any of your own
-modifications if necessary.
-See :doc:`/sysadmins/whatsnew` for details of changes.
+The FastCGI support was removed in OMERO 5.2 and OMERO.web can be deployed
+using WSGI :ref:`unix_omero_web_deployment`. If you deployed OMERO.web using
+WSGI it is highly recommended that you regenerate your
+:ref:`unix_omero_web_deployment`, remembering to merge in any of your own
+modifications if necessary. See :doc:`/sysadmins/whatsnew` for details
+of changes.
 
 If necessary ensure you have set up a regular task to clear out any stale
-OMERO.web session files as described in :ref:`unix_omero_web_maintenance`.
+OMERO.web session files as described in :ref:`unix_omero_web_maintenance`
+or :ref:`windows_omero_web_maintenance`.
```

Comment:
No i didn't meant the references. But the `OMERO.web server configuration` paragraph is a copy-paste of the one inserted above, no?